src/WebJobs.Extensions.OpenAI/Assistants/AssistantBindingConverter.cs

```diff
@@ -88,9 +88,8 @@ public AssistantCreateCollector(IAssistantService chatService, ILogger logger)
 
         public async Task AddAsync(AssistantCreateRequest item, CancellationToken cancellationToken = default)
         {
-            AssistantCreateRequest request = new(item.Id, item.Instructions);
-            await this.chatService.CreateAssistantAsync(request, cancellationToken);
-            this.logger.LogInformation("Created assistant '{Id}'", request.Id);
+            await this.chatService.CreateAssistantAsync(item, cancellationToken);
```

Comment:
Hey @aishwaryabh, do you remember why a new object is being created and passed to CreateAssistantAsync or it is just an oversight?